packages/amplication-client/src/App.tsx

```diff
@@ -97,6 +103,18 @@ function App() {
     ignoreTags: [],
   });
 
+  const closeConfirmation = useCallback(() => {
+    setWorkspaceUpgradeConfirmation(false);
+  }, []);
```

Comment:
What is the benefit of using useCallback while you don't have anything in the dependency array

Reply:
The benefit is that the function will only be declared once instead of every run